### ISSUE TITLE
evaluate roundabout attribute in OSM import (trac #1376)

### ISSUE THREAD

**Issue by behrisch** [eclipse-sumo/sumo#1376]
Migrated from http://sumo.dlr.de/ticket/1376
```json
{
    "status": "new", 
    "changetime": "2014-07-13T12:34:26Z", 
    "description": "", 
    "reporter": "behrisch", 
    "cc": "", 
    "resolution": "", 
    "_ts": "1405254866849253", 
    "component": "net building (NETGEN, NETCONVERT)", 
    "summary": "evaluate roundabout attribute in OSM import", 
    "priority": "major", 
    "keywords": "", 
    "time": "2014-07-13T12:34:26Z", 
    "milestone": "1.0.0", 
    "owner": "", 
    "type": "enhancement"
}
```

**Comment by namdre** [comment]
would only be needed to cross check the --roundabouts.guess heuristc

**Comment by namdre** [comment]
was implemented via #13970
